.changeset/rude-weeks-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': minor
```

Comment:
catalog-node probably likely need a changeset as well.
The version is currently stable so this would be a major version bump requiring this method to be implemented. Can we make refresh optional?